explorer/src/migration/1556898012528-AddTransactionHashStatusToTaskRun.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from 'typeorm'
+
+export class AddTransactionHashStatusToTaskRun1556898012528
+  implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<any> {
+    await queryRunner.query(`
+      ALTER TABLE "task_run" ADD COLUMN "transactionHash" character varying;
```

Comment:
Transaction hashes are always the same size right? Might be an opportunity to use a constraint to catch errors.

Reply:
https://www.pivotaltracker.com/story/show/165842747